fixups/FileRedirectionFixup/PathRedirection.cpp

```diff
@@ -395,7 +427,7 @@ std::wstring RedirectedPath(const normalized_path& deVirtualizedPath, bool ensur
 }
 
 template <typename CharT>
-static path_redirect_info ShouldRedirectImpl(const CharT* path, redirect_flags flags)
+static path_redirect_info ShouldRedirectImpl(const CharT * path, redirect_flags flags)

```

Comment:
>  [](start = 56, length = 1)

Changed spacing here.

Reply:
I did Ctrl+k;Ctrl+d, and seems like it messed up few places